<?xml version="1.0"?>
<launch>

  <arg name="hostname" default="localhost" />

  <!-- Data input only necessary for tests without the robot.
       Can be an entire directory of point clouds, one of which will be selected at random, or a single point cloud file.
	   Training directory only needed if classifier is not working.
   -->
  <arg name="data_dir" default="$(find squirrel_active_exploration)/data/T_01_willow_dataset/" />
  <arg name="train_dir" default="$(find squirrel_active_exploration)/data/training/" />
  <arg name="variance" default="0.5" />
  <arg name="camera_height" default="0.7" />
  <arg name="robot_radius" default="0.22" />
  <arg name="distance_from_center" default="2.0" />
  <arg name="num_locations" default="8" />
  <arg name="tree_resolution" default="0.1" />
  <arg name="reverse_transforms" default="true" />  <!-- is "true" for WILLOW GARAGE dataset -->
  <arg name="occlusions" default="true" />
  <arg name="working_classifier" default="true" />  <!-- is "true" if classifier is working?? -->

  <node name="test_active_exploration_server_robot"
        pkg="squirrel_active_exploration"
        type="test_active_exploration_server_robot"
        output="screen">
       	<param name="data_dir" type="string" value="$(arg data_dir)" />
		<param name="train_dir" type="string" value="$(arg train_dir)" />
    	<param name="variance" type="double" value="$(arg variance)" />
		<param name="camera_height" type="double" value="$(arg camera_height)" />
		<param name="robot_radius" type="double" value="$(arg robot_radius)" />
		<param name="distance_from_center" type="double" value="$(arg distance_from_center)" />
		<param name="num_locations" type="double" value="$(arg num_locations)" />
		<param name="tree_resolution" type="double" value="$(arg tree_resolution)" />
		<param name="reverse_transforms" type="bool" value="$(arg reverse_transforms)" />
		<param name="occlusions" type="bool" value="$(arg occlusions)" />
		<param name="working_classifier" type="bool" value="$(arg working_classifier)" />
  </node>

</launch>
